### DISCUSSION TITLE
Daily 02/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#192] [Daily Sprint]
- Qué hice ayer?
- Qué haré hoy?
- Dificultades e impedimentos

**Comment by NataliaAlvarezIspc** [comment]
El fin de semana estuve haciendo las páginas de login y de restablecer contraseña. También estuve completando algunas tareas de maquetación o diseño. El sábado tuvimos la weekly a la cual me conecté.
Hoy estuve haciendo las tareas de la barra de navegación (agregue carrito y lupa). Ahora continuare con la tarea de  modificación de datos. Al ser un tema nuevo y debido a mi poco conocimiento general, para todo tengo que consultar materiales y ver videos que hacen que me demore mas tiempo.

**Comment by BraianTroncoso** [comment]
El finde repasé un poco mas lo que ví en las clases anteriores de Angular de OPB, también estuve en la weekly ayudando en el diseño del logo como su realización entre todos y hoy vi la clase de Directivas en Angular, voy por la 5 de 17 :,(, es muy pesado pero creo que es de calidad. Un poco lento pero toca cada punto a detalle.
Hoy voy a ver si puedo empezar a realizar las tareas asignadas así podemos finalizar el Spring con éxito.

**Comment by rpgrca** [comment]
Hoy tuve que ir a Capital así que apenas si pude darle una mano a Brian. Voy a tomar algunas de diseño para terminarlas bien rápido. También tuve clase de objetos hoy.
